src/modules/actions/getFilms.js

```diff
@@ -0,0 +1,18 @@
+import { GET_FILMS, API_KEY } from '../constants/constants';
+import getGenre from './getGenre';
+
+const getFilms = () => {
+  return async (dispatch) => {
+    await fetch(`https://api.themoviedb.org/3/movie/popular?api_key=${API_KEY}&language=en-US&page=1`)
+      .then(resp => resp.json())
+      .then(data => dispatch({
```

Comment:
let's move it to the separate action 
```
const getFilms = (results) => ({
        type: GET_FILMS,
        payload: results
      })
```
then 
`dispatch(getFilms(data.results))`

Reply:
changed